wallet-test/src/test/scala/org/bitcoins/wallet/UTXOLifeCycleTest.scala

```diff
@@ -24,7 +24,7 @@ class UTXOLifeCycleTest extends BitcoinSWalletTest {
       .fromString("bcrt1qlhctylgvdsvaanv539rg7hyn0sjkdm23y70kgq")
 
   override def withFixture(test: OneArgAsyncTest): FutureOutcome = {
-    withFundedWalletAndBitcoind(test)
+    withFundedWalletAndBitcoind(test, getBIP39PasswordOpt())
```

Comment:
What is the use case for being able to give a specific bip39 password for a certain test? It would make more sense to just call `getBIP39PasswordOpt` in the fixture (in this case `withFundedWalletAndBitcoind`)

Reply:
The problem is your test case can't access it then if it is generated inside your fixture. I thought about making it a default argument for the methods 

```scala
def withFundedWalletAndBitcoind(test: OneArgAsyncTest, bip39PasswordOpt: Option[String] = getBIP39PasswordOpt()) = ...
```

but I think it is too easy to forget to pass in the bip39 password when you need it.